app/editor/src/features/clips/RequestClip.tsx

```diff
@@ -0,0 +1,167 @@
+      .then((response) => {
+        setIngests(response.data.items);
+      })
+      .catch(); // Errors are handled globally.
+  }, [apiIngest]);
+
+  const onSubmit = async (values: IScheduleModel) => {
+    if (ingest) {
+      // TODO: Test if a duplicate schedule has already been added.
+      try {
+        const response = await apiIngestSchedule.addSchedule({
+          ingestId: ingest.id,
+          requestedById: userInfo?.id,
+          ...values,
+        });
+        if (response.status === 200) {
+          toast.success(
+            `Clip "${values.name}" has successfully been requested.  You will receive a notification when it has been created.`,
+          );
+        }
+      } catch (ex) {
+        // Errors are handled globally.
+      }
+    }
+  };
+
+  return (
+    <styled.RequestClip>
+      <FormPage>
+        <FormikForm onSubmit={onSubmit} initialValues={defaultSchedule}>
+          {({ values, isSubmitting, resetForm }) => (
+            <Col alignContent="center">
+              <p className="text">Request a new clip to be created.</p>
+              <Show visible={!ingests.length}>
+                <p className="error">There are no configured clip ingest services.</p>
+              </Show>
+              <div className="ingest">
+                <Select
+                  name="source"
+                  label="Source"
+                  options={sources}
+                  isDisabled={!sources.length}
+                  onChange={(newValue) => {
+                    const value = (newValue as IOptionItem)?.value;
+                    if (value !== undefined) setSourceId(+value);
+                    else setSourceId(undefined);
+                    setIngest(undefined);
+                    resetForm();
+                  }}
+                />
+                <Select
+                  name="ingest"
+                  label="Ingest"
+                  options={ingestOptions}
+                  isDisabled={!ingestOptions.length}
+                  value={ingestOptions.find((i) => i.value === ingest?.id) ?? ''}
+                  onChange={(newValue) => {
+                    const ingest = ingests.find((i) => i.id === (newValue as IOptionItem)?.value);
+                    setIngest(ingest);
+                    resetForm();
+                  }}
+                />
+                <FormikText
+                  name="name"
+                  label="Name"
+                  disabled={!ingest}
+                  // value={schedule ? schedule.name : ''}
+                  // onChange={(e) => setSchedule({ ...schedule!, name: e.currentTarget.value })}
+                />
+                <Row alignItems="flex-end">
+                  <FormikTimeInput
+                    label="Start At"
+                    name="startAt"
+                    width="7em"
+                    placeholder="HH:MM:SS"
+                    disabled={!ingest}
+                    // value={schedule ? schedule.startAt : ''}
```

Comment:
Assuming this was intentional - but just in case you forgot to uncomment (few of them above and below as well)

Reply:
Thanks for catching this!